compliance-by-design-demo/src/components/KvkForm.js

```diff
@@ -6,6 +6,17 @@ import Container from "react-bootstrap/Container";
 
 import DateControl from "./DateControl";
 
+// Facts that are always assumed to be true.
+const INITIAL_FACTS = {
+  "[verzoek tegemoetkoming in de schade geleden door de maatregelen ter bestrijding van de verdere verspreiding van COVID-19]": true,
+  "[gemeente]": true,
+  "[gedupeerde onderneming]": true,
+  "[Minister van Economische Zaken en Klimaat]": true,
+  "[verzoek om aanvullende uitkering voor levensonderhoud op grond van de Tozo]": true,
+  "[zelfstandige]": true,
```

Comment:
I don't know if [zelfstandige] should always be true.

I.e. if an employee wants to check what a company qualifies for the cases where they are 'zelfstandige' do not apply to the company nor to the person checking, just to the owner.

Reply:
I just added all actors, objects, and recipients to this list. If that's not correct, let me know.

Reply:
I think it makes sense to leave out the actors for now (so [gedupeerde onderneming] and [zelfstandige]). It would be kind of awkward if we say someone gets xxx, but they don't because they do not fall in the relevant category.

Reply:
Just kidding, looks like I forgot that actors are somewhat special-cased, so let's add them as true